pkg/capabilities/triggers/cron_trigger.go

```diff
@@ -0,0 +1,221 @@
+package triggers
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/go-co-op/gocron/v2"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+const cronTriggerID = "cron-trigger@1.0.0"
+
+var cronTriggerInfo = capabilities.MustNewCapabilityInfo(
+	cronTriggerID,
+	capabilities.CapabilityTypeTrigger,
+	"A trigger to schedule workflow execution to run periodically at fixed times, dates, or intervals.",
+)
+
+type cronTriggerConfig struct {
+	// The crontab to evaluate, with second granularity.
+	// The seconds field is required. The time zone will always be automatically set to UTC.
+	Schedule string `json:"schedule"`
+}
+
+type cronTriggerInput struct{}
+
+type CronTriggerMetadata struct{}
+
+type CronTriggerPayload struct {
+	ScheduledExecutionTime string
+}
+
+type CronTriggerResponse struct {
+	capabilities.TriggerEvent
+	Metadata CronTriggerMetadata
+	Payload  CronTriggerPayload
+}
+
+type cronTrigger struct {
+	ch  chan<- capabilities.TriggerRegistrationResponse
+	job gocron.Job
+}
+
+type CronTriggerService struct {
+	capabilities.CapabilityInfo
+	capabilities.Validator[cronTriggerConfig, cronTriggerInput, capabilities.CapabilityResponse]
+	lggr      logger.Logger
+	mu        sync.Mutex
+	scheduler gocron.Scheduler
+	triggers  map[string]cronTrigger
+}
+
+var _ capabilities.TriggerCapability = (*CronTriggerService)(nil)
+var _ services.Service = &CronTriggerService{}
+
+func NewCronTriggerService(lggr logger.Logger) *CronTriggerService {
+	location, _ := time.LoadLocation("UTC")
```

Comment:
You can use time.UTC rather than loading it, this will avoid needing to handle the error